packages/report-server/src/reportBuilder/query/buildPeriodParams.ts

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Moved out of `QueryBuilder` as I was worried that class was gonna get pretty busy